packages/smooth_app/lib/generic_lib/dialogs/smooth_alert_dialog.dart

```diff
@@ -416,6 +416,7 @@ class _SmoothActionFlatButton extends StatelessWidget {
           textStyle: themeData.textTheme.bodyMedium!.copyWith(
             color: themeData.colorScheme.onPrimary,
           ),
+          backgroundColor: Theme.of(context).focusColor,
```

Comment:
Hi @Sudhanva-Nadiger!
I understand the general idea, but unfortunately `focusColor` is not a color we explicitly set in the app (cf. `color_schemes.dart` and `smooth_theme.dart`), meaning that we are just lucky if the color fits our needs.
A more relevant idea would be to use the `secondary`/ `onSecondary`colors, making `secondary` slightly less similar to white.
![Capture d’écran 2023-02-20 à 08 18 23](https://user-images.githubusercontent.com/11576431/220038126-24d978b6-dfdf-4d82-8b67-03271584cc0a.png)

Reply:
I tried other colors 😄none seems to fit, or we can explicitly define the focus color in design constant or colour theme ?

Reply:
Let's assume that focus color has something to do with focus.
Please google "material primary secondary".

Reply:
Okays !

Reply:
<p float="left"> 
<img src="https://user-images.githubusercontent.com/93595710/220290944-f6f427d3-164b-44da-80fc-9b602d8394f5.png" width="170">
<img src="https://user-images.githubusercontent.com/93595710/220290983-d570917d-5c52-476b-b3aa-334be3b1af64.png" width="170">
</p>